File: android/app/src/main/java/com/bitbusters/android/speproject/WIMSPointRenderer.java
package com.bitbusters.android.speproject;

import android.content.Context;

import com.google.android.gms.maps.GoogleMap;
import com.google.android.gms.maps.model.BitmapDescriptorFactory;
import com.google.android.gms.maps.model.MarkerOptions;
import com.google.maps.android.clustering.ClusterManager;
import com.google.maps.android.clustering.view.DefaultClusterRenderer;

/**
 * Created by Stefan on 08/02/2017.
 */

public class WIMSPointRenderer extends DefaultClusterRenderer<WIMSPoint>{

    private GoogleMap mMap;

    public WIMSPointRenderer(Context context, GoogleMap map, ClusterManager<WIMSPoint> clusterManager) {
        super(context, map, clusterManager);
        this.mMap = map;
    }

    @Override
    protected void onBeforeClusterItemRendered(WIMSPoint sp, MarkerOptions markerOptions) {
        markerOptions.icon(BitmapDescriptorFactory.defaultMarker(BitmapDescriptorFactory.HUE_BLUE));
    }





}

File: android/app/src/main/java/com/bitbusters/android/speproject/renderers/WIMSPointRenderer.kt
package com.bitbusters.android.speproject.renderers

import android.content.Context
import com.bitbusters.android.speproject.R
import com.bitbusters.android.speproject.data.WIMSPoint
import com.google.android.gms.maps.GoogleMap
import com.google.android.gms.maps.model.BitmapDescriptorFactory
import com.google.android.gms.maps.model.MarkerOptions
import com.google.maps.android.clustering.ClusterManager
import com.google.maps.android.clustering.view.DefaultClusterRenderer

/**
 * Created by mihajlo on 04/07/17.
 */
class WIMSPointRenderer(context: Context, val mMap: GoogleMap, clusterManager: ClusterManager<WIMSPoint>) :
        DefaultClusterRenderer<WIMSPoint>(context, mMap, clusterManager) {

    override fun onBeforeClusterItemRendered(sp: WIMSPoint, markerOptions: MarkerOptions) {
        markerOptions.icon(BitmapDescriptorFactory.fromResource(R.drawable.ic_wims_marker))
    }
}